python/hail/linalg/matrix.py

```diff
@@ -20,6 +23,42 @@ def read(cls, path):
         return cls(Env.hail().distributedmatrix.BlockMatrix.read(
             hc._jhc, path))
 
+    @staticmethod
+    @handle_py4j
+    def _from_numpy_matrix(numpy_matrix, block_size):
+        """Create a BlockMatrix from the given numpy matrix.
+
+        Notes
+        -----
+
+        This method is being used to create a BlockMatrix for testing BlockMatrix's entries_table() method.
```

Comment:
How about something more durable like: "This method is useful for creating small block matrices when writing tests. It is not efficient for large matrices."